examples/hello_isahc/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "hello_isahc"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

```suggestion
publish = false
```

Seems sensible here.